packages/mqttsuite/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/SNodeC/mqttsuite
+TERMUX_PKG_DESCRIPTION="MQTTSuite: A lightweight MQTT Integration System"
+TERMUX_PKG_LICENSE="MIT OR LGPL-3.0-or-later"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
+TERMUX_PKG_MAINTAINER="VolkerChristian"
+TERMUX_PKG_VERSION=1.0
+TERMUX_PKG_SRCURL="git+https://github.com/SNodeC/mqttsuite.git"
+TERMUX_PKG_DEPENDS="fmt,snode.c"
+TERMUX_PKG_FORCE_CMAKE=true
+
```

Comment:
Please remove this additional newline.

The linter will flag it as a codestyle error.
![image](https://github.com/user-attachments/assets/1040a38c-7ed4-4d0a-b42b-1c738a34ae1a)